### PR TITLE
PE-32 list detail base

### PR DESCRIPTION
**Pull request by alirni**

### CODE REVIEW

**Review by alimd — CHANGES_REQUESTED** [review]
We must change structure for page-entity together

File: README.md [hunk 1]
```diff
@@ -1,38 +1,21 @@
 # \<list-detail\>
 
-
+iOrder PWA Elements List Detail
 
 ## Install the Polymer-CLI
 
 First, make sure you have the [Polymer CLI](https://www.npmjs.com/package/polymer-cli) installed. Then run `polymer serve` to serve your application locally.
```
Comment (alimd) [inline README.md]: Please remove below this line

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	],
+  "dependencies": {
+    "polymer": "Polymer/polymer#^1.4.0"
+  },
+  "devDependencies": {
+    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
+    "web-component-tester": "^4.0.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+  }
```
Comment (alimd) [inline bower.json:49]: Please add resolutions

**Review by alimd — CHANGES_REQUESTED** [review]
Please switch to polymer 2

**Comment by alimd** [comment]
Convert to space 😠

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  "private": true,
+  "ignore": [
+		"**/.*",
+		"node_modules",
+		"bower_components",
+		"test",
+		"tests"
+	],
+  "dependencies": {
+    "polymer": "Polymer/polymer#2.0-preview",
```
Comment (alirni) [inline bower.json:28]: fix tab

**Pull request closed without merging**